### PR TITLE
EWPP-2286: Add Description field for Person content type.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: modules/oe_content_person/config/install/field.storage.node.oe_person_description.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+langcode: en
```
Comment (22Alexandra) [inline modules/oe_content_person/config/install/field.storage.node.oe_person_description.yml:1]: Why not using the body field storage as we do for the rest of the CTs?

File: modules/oe_content_person/oe_content_person.post_update.php [hunk 2]
```diff
@@ -72,3 +75,56 @@ function oe_content_person_post_update_20003(): void {
...
+  // View display configuration to update.
+  $view_display_values = $storage->read('core.entity_view_display.node.oe_person.default');
+  $view_display_values['_core']['default_config_hash'] = Crypt::hashBase64(serialize($view_display_values));
+  $view_display = EntityViewDisplay::load($view_display_values['id']);
+  if ($view_display) {
+    $updated_view_display = \Drupal::entityTypeManager()
+      ->getStorage($view_display->getEntityTypeId())
+      ->updateFromStorageRecord($view_display, $view_display_values);
+    $updated_view_display->save();
+  }
```
Comment (22Alexandra) [inline modules/oe_content_person/oe_content_person.post_update.php]: We should update the view display only in oe_theme where we take care of the rendering.

**Review by 22Alexandra — CHANGES_REQUESTED** [review]
There is a test failing to be fixed.

File: modules/oe_content_person/config/install/field.storage.node.oe_person_description.yml (hunk 1 above)
Comment (sergepavle) [reply modules/oe_content_person/config/install/field.storage.node.oe_person_description.yml:1]: I would not reuse the body field. Initially, that field was used for the main content of content type. But description field in person content type is more used as additional content.

File: modules/oe_content_person/oe_content_person.post_update.php (hunk 2 above)
Comment (sergepavle) [reply modules/oe_content_person/oe_content_person.post_update.php]: removed.

File: modules/oe_content_person/config/install/field.storage.node.oe_person_description.yml (hunk 1 above)
Comment (22Alexandra) [reply modules/oe_content_person/config/install/field.storage.node.oe_person_description.yml:1]: Alright.

**Review by 22Alexandra — DISMISSED** [review]

**Review by nagyad — DISMISSED** [review]

**Review by nagyad — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**